src/frontend/typer/vilkår.ts

```diff
@@ -1,42 +1,62 @@
-import { INøkkelPar } from './common';
 import { IPeriode } from './periode';
 import { IGrunnlagPerson, PersonType } from './person';
 import { IRestVedtakBegrunnelse, VedtakBegrunnelseType } from './vedtak';
 import { FeltState } from '../familie-skjema/typer';
 import { BehandlingSteg, BehandlingStegStatus } from './behandling';
 
 export enum Resultat {
+    IKKE_OPPFYLT = 'IKKE_OPPFYLT',
+    OPPFYLT = 'OPPFYLT',
+    IKKE_VURDERT = 'IKKE_VURDERT',
+}
+
+export enum RestResultat {
+    IKKE_OPPFYLT = 'IKKE_OPPFYLT',
+    OPPFYLT = 'OPPFYLT',
+    IKKE_VURDERT = 'IKKE_VURDERT',
     NEI = 'NEI',
     JA = 'JA',
     KANSKJE = 'KANSKJE',
 }
 
-export const resultatTilUi = (resultat: Resultat) => {
+export const uiResultat: Record<Resultat, string> = {
+    OPPFYLT: 'Oppfylt',
+    IKKE_OPPFYLT: 'Ikke oppfylt',
+    IKKE_VURDERT: 'Ikke vurdert',
+};
+
+export const resultater: Record<Resultat, string> = {
+    OPPFYLT: 'JA',
+    IKKE_OPPFYLT: 'NEI',
+    IKKE_VURDERT: 'Kanskje',
+};
+
+export const migrerResultatTilUi = (resultat: RestResultat): Resultat => {
```

Comment:
Må bare klargjøre litt for meg selv her. Er planen 4 PR'er? (altså litt utvidet expand contract, siden vi må håndtere "begge veier") 
1. Backend som støtter mottak av begge ut, men returnerer fortsatt kun V1-format.
2. Frontend sender kun V2-format, men støtter mottak av begge
3. Backend returnerer kun V2-format og forholder seg nå kun til V2.
4. Fjerne mottak av V1 frontend. Frontend forholder seg nå kun til V2.

Reply:
haha, jepp må bli sånn frontend og for e2e 🤯